### DISCUSSION TITLE
Command line 语音识别是否支持预测标点

### DISCUSSION THREAD

**Discussion by ultrasev** [PaddlePaddle/PaddleSpeech#1250] [Q&A] [answered]
刚试用了一下command line 转录语音，感觉很棒。 有一个小问题， 转录结果中没有看到标点，是不是我这边参数没有设置好？ 

![](https://s2.loli.net/2021/12/30/69YTPsGLwRJUVK5.png)

**Comment by yt605155624** [comment, marked as answer]
标点预测是额外的后处理
https://github.com/PaddlePaddle/PaddleSpeech/tree/develop/demos/punctuation_restoration

**Comment by ultrasev** [comment]
感谢大佬，太棒了。
